src/main/java/com/mattmalec/pterodactyl4j/client/entities/impl/DatabaseManagerImpl.java

```diff
@@ -0,0 +1,80 @@
+package com.mattmalec.pterodactyl4j.client.entities.impl;
+
+import com.mattmalec.pterodactyl4j.PteroAction;
+import com.mattmalec.pterodactyl4j.client.entities.ClientServer;
+import com.mattmalec.pterodactyl4j.client.entities.Database;
+import com.mattmalec.pterodactyl4j.client.managers.DatabaseAction;
+import com.mattmalec.pterodactyl4j.client.managers.DatabaseManager;
+import com.mattmalec.pterodactyl4j.requests.PteroActionImpl;
+import com.mattmalec.pterodactyl4j.requests.Route;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class DatabaseManagerImpl implements DatabaseManager {
+
+    private final ClientServer server;
+    private final PteroClientImpl impl;
+
+    public DatabaseManagerImpl(ClientServer server, PteroClientImpl impl) {
+        this.server = server;
+        this.impl = impl;
+    }
+
+    @Override
+    public DatabaseAction createDatabase() {
+        return new CreateDatabaseImpl(server, impl);
+    }
+
+    @Override
+    public PteroAction<Database> getDatabase(String databaseId) {
+        return PteroActionImpl.onRequestExecute(impl.getP4J(),
+                Route.Databases.LIST_DATABASES.compile(server.getIdentifier()), (response, request) -> {
+                    JSONObject json = response.getObject();
+                    JSONObject parsedJson = new JSONObject();
+                    JSONArray dataArray = json.getJSONArray("data");
+
+                    for (int i=0; i < dataArray.length();) {
+                        String jsonDbId = json.getJSONArray("data").getJSONObject(i).getJSONObject("attributes").getString("id");
+                        if (Objects.equals(jsonDbId, databaseId)){
```

Comment:
Also for your own reference, you shouldn't be using Objects#equals here. The following would be fine.

```suggestion
                        if (jsonDbId.equals(databaseId)) {
```